### PR TITLE
Update Optics Devices For Cooling

### PR DESCRIPTION
**Pull request by nrwslac**
## Description
- Added cooling components to class `XOffsetMirrorBend`
- Removed class `CoatingState`
- class `XOffsetMirrorState` removed `CoatingState` inheritance.
- `XOffsetMirrorState` components reordered using decorator, coating component added in class.
- New class `XOffsetMirrorStateCool` created for offset mirrors with state and cooling.
- update spectrometer class `Mono` to use cooling PVs as implemented.

## Motivation and Context

New cooling pressure and flow meters installed, update devices to reflect.

## How Has This Been Tested?

Opened Typhos screens for:

`MR2K4:KBO`
`MR1K4:SOMS`
`MR1K1:BEND`
`SP1K1:MONO`

Using  this branch. Checked readbacks; See screenshot for example.

## Where Has This Been Documented?

Documented here in pre-release notes.

## Screenshots :
mirror-fwm-00

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by ZLLentz** [comment]
This is a pretty good clean-up :+1:

**Review by klauer — APPROVED** [review]
Changes look good to me 👍 

The changelog is acronym soup, but I suppose that's the nature of the game...

You indicated the typhos screens work and have checked all the boxes for a good PR here, so I think this is good to go.

File: docs/source/upcoming_release_notes/1106-optics_cooling.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Device Updates
+--------------
+- `XOffsetMirrorBend` gets cooling PVs: `FWM:*_RBV` and `PRSM:*_RBV`
+  `KBOMirrorHE` set PVs to `FWM` and `PRSM` to match ccc.
+  `Mono` set PVs to `FWM` and `PRSM` to match ccc.
+
+New Devices
+-----------
+- `XOffsetMirrorStateCool` created for offset mirrors with state and cooling.
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1106-optics_cooling.rst:20]: 😎

File: docs/source/upcoming_release_notes/1106-optics_cooling.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- removed `CoatingState` class, used `@reorder_components` instead.
+
+Contributors
+------------
+- N/A
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1106-optics_cooling.rst:32]:
```suggestion
- nrwslac
```

**Review by ZLLentz — APPROVED** [review]
Looks good to me! My only suggestion is to add yourself to the contributors list here

**Review by tangkong — APPROVED** [review]
What they said 👍

File: docs/source/upcoming_release_notes/1106-optics_cooling.rst (hunk 1 above)
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1106-optics_cooling.rst:20]: keep it cool.

File: docs/source/upcoming_release_notes/1106-optics_cooling.rst (hunk 2 above)
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1106-optics_cooling.rst]: updated.

**Pull request merged**